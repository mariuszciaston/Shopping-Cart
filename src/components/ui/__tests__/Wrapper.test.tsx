import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import Wrapper from "../Wrapper";

describe("Wrapper Component", () => {
  it("renders the Wrapper component correctly", () => {
    const { getByText } = render(
      <Wrapper>
        <div>Test Child</div>
      </Wrapper>,
    );

    const childElement = getByText("Test Child");
    expect(childElement).toBeInTheDocument();
  });

  it("matches the snapshot", () => {
    const { asFragment } = render(
      <Wrapper>
        <div>Test Child</div>
      </Wrapper>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
