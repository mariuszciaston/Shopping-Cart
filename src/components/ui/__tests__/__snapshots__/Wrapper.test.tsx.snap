// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Wrapper Component > matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex h-[calc(100vh)] flex-col"
    id="wrapper"
  >
    <div>
      Test Child
    </div>
  </div>
</DocumentFragment>
`;
